### PR TITLE
DOC-1381: Onboarding Parking Services

### PR DESCRIPTION
**Pull request by martapederiva**
## Link to JIRA ticket

https://hackney.atlassian.net/browse/DOC-1381

### *What is the problem we're trying to solve*

Adding the necessary configuration to onboard Parking Services to use DES

### *What changes have we introduced*

Config JSON files

### CODE REVIEW

**Review by bogdan-zaharia — COMMENTED** [review]
Looks good, left a few comments

File: EvidenceApi/DocumentTypes.json [hunk 1]
```diff
@@ -352,5 +352,23 @@
...
       }
     ]
-  }
+  },
+  {
+    "name": "Hackney Parking Services",
+    "id": "11",
+    "documentTypes": [
+      {
+        "id": "vehicle-photos",
```
[the remark this answers is not in the document]
Comment (martapederiva) [reply EvidenceApi/DocumentTypes.json]: addressed in [e43e475](https://github.com/LBHackney-IT/evidence-api/pull/192/commits/e43e4756bed1ec745c019a368df06a8812ddc116)

File: EvidenceApi/StaffSelectedDocumentTypes.json [hunk 2]
```diff
@@ -934,5 +934,23 @@
...
+    "documentTypes": [
+      {
+        "id": "vehicle-photos",
+        "title": "Photograph",
+        "description": "",
+        "enabled": true
+      },
+      {
+        "id": "other-council-letter",
+        "title": "Council letter",
```
[the remark this answers is not in the document]
Comment (martapederiva) [reply EvidenceApi/StaffSelectedDocumentTypes.json]: addressed in [e43e475](https://github.com/LBHackney-IT/evidence-api/pull/192/commits/e43e4756bed1ec745c019a368df06a8812ddc116)

File: EvidenceApi/StaffSelectedDocumentTypes.json [hunk 3]
```diff
@@ -941,13 +941,13 @@
     "documentTypes": [
       {
         "id": "vehicle-photos",
-        "title": "Photograph",
+        "title": "Photograps of vehicle",
```
Comment (bogdan-zaharia) [inline EvidenceApi/StaffSelectedDocumentTypes.json:944]: Typo in "Photograps"

**Review by bogdan-zaharia — COMMENTED** [review]
Left another comment

File: EvidenceApi/StaffSelectedDocumentTypes.json (hunk 3 above)
Comment (martapederiva) [reply EvidenceApi/StaffSelectedDocumentTypes.json]: Addressed in [7167a4f](https://github.com/LBHackney-IT/evidence-api/pull/192/commits/7167a4f1f541071dc1b863c4c5d8a016c3c54033)

**Review by bogdan-zaharia — APPROVED** [review]
LGTM

**Pull request merged**